shenyu-plugin/shenyu-plugin-auth/src/main/resources/application.properties

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+casdoor.endpoint=http://localhost:8000
```

Comment:
how to modify this？

Reply:
Casdoor default endpoint is localhost:8000,if it changed,we need change it here

Reply:
who modify here？ Not everyone is compiling source code or modifying it.


Maybe the Casdoor  initialization of Casdoor should be inside the handlerPluginData method.

you can refer ： rateLimiter plugin or else plugin

Reply:
ok,i will cahnge it

Reply:
this file maybe remove？

Reply:
ok